stacktrace_translator.py

```diff
@@ -297,6 +300,19 @@ def collect_modules(config, branch, rev, platform, dbgsymdir = None):
 	log.info('\t[OK]')
 	return dbgfile, modules
 
+
+def detect_exebase(infolog):
+	match = re.search(RE_EXEBASE, infolog, re.MULTILINE)
+	global EXEBASE
+	EXEBASE = int(match.group(4), 16) if match else 0
+
+def update_base(module, addresses, tempfile):
+	pe = pefile.PE(name=tempfile.name, fast_load=True)
+	image_base = pe.OPTIONAL_HEADER.ImageBase
+	load_base = 0 if module.endswith('.dll') else EXEBASE
+	return [hex(int(x, 16) - load_base + image_base) for x in addresses]
```

Comment:
`return [hex(int(x, 16) & 0xFFFFFFFF - load_base + image_base) for x in addresses]`
could work for current broken crashhandler, but it would be unexplained hax.